templates/email/default/templated_email_alert-update.txt

```diff
@@ -0,0 +1,29 @@
+Subject: New [% site_name %] council update on report: '[% report.title %]'
```

Comment:
Same as above here, I think we should say "New [% site_name %] council update**s** on report"